raiden/ui/console.py

```diff
@@ -216,27 +224,6 @@ def register_token(self, token_address_hex):
         self._raiden.register_channel_manager(channel_manager.address)
         return channel_manager
 
-    def ping(self, peer_address_hex, timeout=4):
```

Comment:
There needs to be another way to enable user pings. We could for example allow for `nonce=0` pings for users.